app/app/views/company/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Index Company Branches</h1>
+
+<ul>
+<% @companys.each do |company| %>
+  <li>
+  <%= link_to @company.name, company_path %>
+  </li>
+<% end %>
+</ul>
```

Comment:
fix the identation of two spaces and Add a blank line on the of this file, please!